deepwell/src/models/sea_orm_active_enums.rs

```diff
@@ -23,7 +39,7 @@ pub enum PageRevisionType {
     Debug, Copy, Clone, PartialEq, Eq, EnumIter, DeriveActiveEnum, Serialize, Deserialize,
 )]
 #[sea_orm(rs_type = "String", db_type = "Enum", enum_name = "user_type")]
-#[serde(rename_all = "camelCase")]
+#[serde(rename_all = "kebab-case")]
```

Comment:
Why kebab-case? And how does it affect this struct any differently from camelCase, considering all the enums are one word?

Reply:
This is one of the things the autogeneration tool removes and I need to re-add. I guess it was originally camelCase though in practice all the enum values are intended to be kebab-case (but this hasn't come up since they're all one word).

If you're asking why this is necessary at all, the issue is otherwise it'll be stored as `Foo` and `Bar` not `foo` and `bar`.